tools/triage_tests.py

```diff
@@ -102,7 +102,6 @@ def eventFilter(self, receiver, event):
             self.window.keyPressEvent(event)
             return True
         else:
-            return False
```

Comment:
Was this a workaround for something?

Reply:
No. The second `return` was introduced later in https://github.com/matplotlib/matplotlib/commit/9a9f68f78ad6f0a736b5269eac010b1de26ff1dc.

Essentially, the effect should be identical. By returning `False` the filter decides that the event should not be handled. The later code defers that decision to the base class, which is formally the correct way of handling this (http://doc.qt.io/archives/qt-4.8/qobject.html#eventFilter). However, `QObject` also just returns false.

I don't mind which way we use. It just should not be two return statements.

Reply:
This chases back one more commit to 6062c2f22f0ab5901d4abe2e5e3c123b525424b4 looks like it came in with two returns